index.html

```diff
@@ -1,34 +1,56 @@
 
-  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+    <link rel="stylesheet" href="css/index.css">
+    <script src="js/box-colors.js"></script>
 
 </head>
 
 <body>
     <div class="container">
-   
-        <section class="top-content">
-            <div class="text-container">
-                <h2>The Future</h2>
-                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
-            </div>
-            <div class="text-container">
-                <h2>The Past</h2>
-                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
-            </div>
-        </section>
-        
+        <header>
+            <section class="nav-content">
+                <img src="img/lambda-black.png" alt="Lambda Logo">
+
+                <nav class="nav-bar">
+                    <a href="index.html" class="nav-item">Home</a>
+                    <a href="about.html" class="nav-item">About</a>
+                    <a href="#" class="nav-item">Products</a>
+                    <a href="#" class="nav-item">Blog</a>
+                    <a href="#" class="nav-item">Contact</a>
+                </nav>
+            </section>
+
+            <section class="top-content">
+                <img src="img/jumbo.jpg" class="jumbo-image">
+
+                <div class="text-container">
+                    <h2>The Future</h2>
+                    <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt
+                        efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas
+                        tincidunt. Nam condimentum venenatis magna eget finibus.</p>
+                </div>
+                <div class="text-container">
+                    <h2>The Past</h2>
+                    <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt
+                        efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas
+                        tincidunt. Nam condimentum venenatis magna eget finibus.</p>
+                </div>
+            </section>
+
+        </header>
+
         <section class="middle-content">
-        
-            <h2>Why Did It Have To Be Boxes...</h2>
-            
+
+            <h2 class="indent">Why Did It Have To Be Boxes...</h2>
+
             <div class="boxes">
```

Comment:
The way you got these boxes colored is really interesting! I will need to clone your project, as the colors are not showing up in the git preview